wordpress-mobile/gutenberg-mobile
# Fix react hook error on block selection

Fixes #1009

This PR cleans up our RN build and makes sure we don't include react twice. The actual fix is executing `gutenberg/node_modules` using `resolver.blacklistRE` in `rn-cli.config.js`

To test:
- Checkout this branch locally
- Run `cd gutenberg && npm install`
- Boot the demo app on android `yarn start:reset` and `yarn android`
- Select an image block
- The app should not crash
